src/hb-set-private.hh

```diff
@@ -324,7 +324,7 @@ struct hb_set_t
   static const unsigned int SHIFT = 5;
   static const unsigned int BITS = (1 << SHIFT);
   static const unsigned int MASK = BITS - 1;
-  static const unsigned int ELTS = (MAX_G + 1 + (BITS - 1)) / BITS;
+  static const unsigned int ELTS = (MAX_G + 1) / BITS;
```

Comment:
FWIW, old cold was implementing ceil(), which is correct, since it does not make assumption on `MAX_G + 1` being divisible by BITS.